coprocessor/fhevm-engine/scheduler/src/dfg/scheduler.rs

```diff
@@ -764,58 +385,270 @@ fn partition_components(
     Ok(())
 }
 
-type TaskResult = Result<(SupportedFheCiphertexts, Option<(i16, Vec<u8>)>)>;
+fn re_randomise_transaction_inputs(
+    inputs: &mut HashMap<Handle, Option<DFGTxInput>>,
+    transaction_id: &Handle,
+    gpu_idx: usize,
+    cpk: tfhe::CompactPublicKey,
+) -> Result<()> {
+    let transaction_rerandomisation_domain_separator = *b"TFHE-w.r";
+    let compact_public_encryption_domain_separator = *b"TFHE.Enc";
```

Comment:
checked in the spec this should be `TFHE_Enc` we used to have an incorrect value in our tests, the one I indicate should be in line with the latest spec